versioned_docs/version-0.2.0/contribute-to-kubeslice-docs.md

```diff
@@ -0,0 +1,52 @@
+# Contribute to KubeSlice documentation
```

Comment:
We follow Title case and add gerund instead of an imperative verb. 

The title can be `Contributing to KubeSlice Documentation`